### ISSUE TITLE
Trace calls/syscalls

### ISSUE THREAD

**Issue by Maijin** [radareorg/cutter#1023]
Trace calls/syscalls in emulation, debug mode using `dcs*` etc.

**Comment by xarkes** [comment]
Removed from Debugger project as it is not important for having a working debugger.

**Comment by ITAYC0HEN** [comment]
I don't think that the Debug project contains only things that required for debugger to work, same as issues in X-refs project don't imply that x-ref widget doesn't work

**Comment by XVilka** [comment]
Probably makes sense to make a separate project called "Debugger
Improvemeints", and move them here.
